test/loadTest/updates_time_measure.sh.sh

```diff
@@ -0,0 +1,92 @@
+﻿#!/bin/bash
+# -*- coding: latin-1 -*-
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# Author: Ivan Arias Leon
+
+# variables
+HOST="localhost"
+PORT="1026"
+NGSI_VERSION="v1"
+UPDATE_TOTAL=100
+SUBSC=false
+SUBSC_REFERENCE="http://localhost:1028/notify"
```

Comment:
In shell, everything is treated as strings (well, more or less), so citation marks aren't necessary

    HOST=localhost

is valid too

NTC